src/components/List/RowOptionsLink.js

```diff
@@ -3,16 +3,27 @@ import PropTypes from "prop-types";
 import styled from "styled-components";
 import classnames from "classnames";
 
-import { Column } from "../Grid";
+import { Column, Row } from "../Grid";
 import { Link } from "../Text";
 import { mediumAndUp } from "../../theme/mediaQueries";
 import constants from "../../theme/constants";
+import spacing from "../../theme/spacing";
 
-const OptionsContent = styled(Column)`
+import {
+  CHEVRON_ICON_PADDING,
+  CHEVRON_ICON_SIZE,
+  ROW_DATE_MEDIUM_WIDTH,
+  ROW_DATE_SMALL_WIDTH
+} from "./constants";
+
+const CHEVRON_ICON_WIDTH = `${CHEVRON_ICON_SIZE}px + ${CHEVRON_ICON_PADDING} + ${CHEVRON_ICON_PADDING}`;
```

Comment:
We shouldn't assign a concatenated string composed of mathematical operations as a constant value

Reply:
The idea was to add some clarification for the reason adding this numbers. Removed `CHEVRON_ICON_WIDTH`.

Reply:
The clarification is good, but this `CHEVRON_ICON_WIDTH` variable should have a numerical value, or px count value, rather than being a concatenated string composed of mathematical operations